.gitmodules

```diff
@@ -49,3 +49,6 @@
 [submodule "src/deps/imgui"]
 	path = src/deps/imgui
 	url = https://github.com/ocornut/imgui.git
+[submodule "src/deps/tinyobjloader"]
+	path = src/deps/tinyobjloader
+	url = https://github.com/syoyo/tinyobjloader.git
```

Comment:
Just a question -- what tinyobjloader can do but Assimp's OBJ loader can't?

Reply:
Assimp seemed too high-level and more tuned to rendering. I just needed a simple parser to get access to the vertices and faces. I looked into using Assimp and it looked a lot more complex. I also need to preserve face order so that it matches with semantic data which assumes face order.